tasmota/tasmota.ino

```diff
@@ -295,6 +295,9 @@ struct TasmotaGlobal_t {
   RulesBitfield rules_flag;                 // Rule state flags (16 bits)
 
   StateBitfield global_state;               // Global states (currently Wifi and Mqtt) (8 bits)
+#ifdef ESP32
+  bool camera_initialized;                  // For esp32-webcam, to be used in discovery
+#endif  // ESP32
```

Comment:
There is a reason why variables within a struct are located by size. Pls select another location for this bool. Most obvious location is below `ota_factory`. In that case there is no need for #ifdef ESP32 either as it is already there.